src/components/molecules/AppDatepicker.vue

```diff
@@ -103,98 +100,92 @@ const dayClassObj = (day: TDay) => ({
 })
 
 const nextMonth = () => {
-  currDateCursor.value = addMonths(currDateCursor.value, 1)
+  currDateCursor.value = currDateCursor.value.add(1, 'M')
 }
 
 const prevMonth = () => {
-  currDateCursor.value = addMonths(currDateCursor.value, -1)
+  currDateCursor.value = currDateCursor.value.add(-1, 'M')
 }
 
 const setSelectedDate = (day: TDay) => {
   selectedDate.value = day.date
-  emit('update:modelValue', format(selectedDate.value, 'yyyy-MM-dd'))
+  emit('update:modelValue', selectedDate.value.format('YYYY-MM-DD'))
   // change calendar to correct month if they select previous or next month's days
   if (!day.isCurrentMonth) {
-    const selectedMonth = getMonth(selectedDate.value)
-    currDateCursor.value = setMonth(currDateCursor.value, selectedMonth)
-  }
-  hideCalendar()
-}
-
-const handleOutsideClick = (e: Event) => {
-  e.stopPropagation()
-
-  if (!calendar.value?.contains(e.target as Node)) {
-    hideCalendar()
+    const selectedMonth = selectedDate.value.month()
+    currDateCursor.value = currDateCursor.value.month(selectedMonth)
   }
 }
-
-useEventListener(document, 'click', handleOutsideClick)
-useEventListener(document, 'touchstart', handleOutsideClick)
 </script>
 <template>
-  <div
-    ref="calendar"
-    class="relative"
-  >
+  <AppPopover>
     <input
       :value="formatedDate"
       class="border border-tansparent rounded-md outline-none w-full py-2 px-3 focus:border-gray-400"
       readonly
-      @focus="showCalendar"
-    >
-    <div
-      v-if="isShow"
-      class="calendar-container"
     >
-      <header class="flex items-center justify-around col-span-7">
-        <button
-          class="icon-btn text-sm"
-          @click="prevMonth"
-        >
-          <uil:angle-double-left />
-        </button>
-        <span class="p-2">
-          {{ curMonth }} {{ curYear }}
-        </span>
-        <button
-          class="icon-btn text-sm"
-          @click="nextMonth"
-        >
-          <uil:angle-double-right />
-        </button>
-      </header>
+    <template #content>
       <div
-        v-for="dayLabel in DAY_LABELS_TW"
-        :key="dayLabel"
-        class="flex items-center justify-center"
+        class="calendar-container"
       >
-        {{ dayLabel }}
-      </div>
-      <div
-        v-for="(day, index) in dates"
-        :key="index"
-        class="rounded"
-        :class="dayClassObj(day)"
-      >
-        <button
-          class="date-btn"
+        <header class="flex items-center justify-around col-span-7">
+          <button
+            class="icon-btn text-sm"
+            @click="prevMonth"
+          >
+            <uil:angle-double-left />
+          </button>
+          <span class="p-2">
+            {{ curMonth }} {{ curYear }}
+          </span>
+          <button
+            class="icon-btn text-sm"
+            @click="nextMonth"
```

Comment:
```suggestion
            @click.stop="nextMonth"
```
防止切換月份後 Popover 被關起來